solr/core/src/test/org/apache/solr/cli/TestExportTool.java

```diff
@@ -218,6 +231,66 @@ public void testVeryLargeCluster() throws Exception {
     }
   }
 
+  @Test
+  public void testWithBasicAuth() throws Exception {
+    String COLLECTION_NAME = "secureCollection";
+    String USER = "solr";
+    String PASS = "SolrRocksAgain";
+    final String SECURITY_JSON =
+        Utils.toJSONString(
+            Map.of(
+                "authorization",
+                Map.of(
+                    "class",
+                    RuleBasedAuthorizationPlugin.class.getName(),
+                    "user-role",
+                    singletonMap(USER, "admin"),
+                    "permissions",
+                    singletonList(Map.of("name", "all", "role", "admin"))),
+                "authentication",
+                Map.of(
+                    "class",
+                    BasicAuthPlugin.class.getName(),
+                    "blockUnknown",
+                    true,
+                    "credentials",
+                    singletonMap(USER, getSaltedHashedValue(PASS)))));
+
+    configureCluster(2)
```

Comment:
Can the behavior be tested with one node instead of two?  Authentication seems unrelated to the size of the cluster.